appserver/tests/appserv-tests/devtests/admin/cli/src/admin/AdminObjectTest.java

```diff
@@ -371,7 +371,7 @@ private void testDeleteAdminObjectInStandaloneInstance() {
         reportResultStatus(testName, result);
         //reportExpectedResult(testName, result, STANDALONE_INSTANCE_NAME, INSTANCE1_NAME, INSTANCE2_NAME);
         reportExpectedResult(testName, result, "Administered object resource-1 deleted.");
-        reportUnexpectedResult(testName, result, SERVER, CLUSTER_NAME);
+        //reportUnexpectedResult(testName, result, SERVER, CLUSTER_NAME);
```

Comment:
Please add a comment with the issue number that Deepak raised for this problem. Do that for all of the commented tests.